src/components/ThemeMode/ThemeMode.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from './themeMode.module.scss';
+import { useRef } from 'react';
+import { getIsDarkMode, toggleDarkMode } from '../../config/themeMode/themeModeStorage';
+import IconButton from '../IconButton/IconButton';
+import clsx from 'clsx';
+import { FiSun, FiMoon } from 'react-icons/fi';
+
+const ThemeMode = () => {
+  const lightModeIconRef = useRef<HTMLButtonElement>(null);
+  const darkModeIconRef = useRef<HTMLButtonElement>(null);
+  const isDarkModeFromLocalStorage = getIsDarkMode();
+  isDarkModeFromLocalStorage && document.body.classList.add('theme-dark');
+
+  const handleOnclick = () => {
```

Comment:
handleOnClick is still not changed - handleOn(What??)Click ?

Name handlers the way that when you read them - you have an idea what theire for.
i.e hadleThemeButtonClick / handleToggleTheme / handleThemeSwitch